solr/core/src/test/org/apache/solr/core/TestLazyCores.java

```diff
@@ -896,7 +896,7 @@ private LocalSolrQueryRequest makeReq(SolrCore core, String... paramPairs) {
   }
 
   private static String makePath(String... args) {
-    return String.join(File.separator, args);
+    return String.join(FileSystems.getDefault().getSeparator(), args);
```

Comment:
This isn't wrong but I think Path.of(.....).toString() would be more idiomatic for its purpose.  And looking at it's callers, just 4 right in a row, you could inline this if you wish (I would).